### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Liam0**
No Cash/license update when you purchase license.
Press escape to get rid of the annoying storagebox.
Confirmation before selling vehicles out of the garage.

### CODE REVIEW

**Comment by tkcjesse** [comment]
ERROR: Possible missing white space after if detected at ../Framework/Altis_Life.Altis/core/functions/fn_keyHandler.sqf Line number: 55

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf [hunk 1]
```diff
@@ -50,6 +50,14 @@ if (life_container_active) then {
         case 57: {
             [] spawn life_fnc_placestorage;
         };
+        //Esc
+        case 1: {
+            if(!isnull life_container_activeObj) then {
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: make if (

Comment (Liam0) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: Thank you :)

**Comment by tkcjesse** [comment]
Have you tested all of this?

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf [hunk 2]
```diff
@@ -50,6 +50,14 @@ if (life_container_active) then {
         case 57: {
             [] spawn life_fnc_placestorage;
         };
+        //Esc
+        case 1: {
+            if (!isnull life_container_activeObj) then {
```
Comment (Jawshy) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: `isNull`*

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf [hunk 3]
```diff
@@ -50,6 +50,14 @@ if (life_container_active) then {
...
+                deletevehicle life_container_activeObj;
```
Comment (Jawshy) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: `deleteVehicle`

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf (hunk 2 above)
Comment (tkcjesse) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: Personally. I'd just like to say I sometimes have the urge to throat punch you...

Comment (Jawshy) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: For those who don't know, this person is a moderator on the Altis Life RPG Forums.

Comment (tkcjesse) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: What's yer point?!

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf [hunk 4]
```diff
@@ -50,6 +50,14 @@ if (life_container_active) then {
...
+                life_container_active = false;
+                life_container_activeObj = ObjNull;
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: objNull*

I just wanted to feel like I was part of the group

Comment (tkcjesse) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: This guy.

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf (hunk 2 above)
Comment (Jawshy) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: Comments like the above just aren't what would be expected of a moderator. Surely you wouldn't make such comments on the forums.

**Comment by Jawshy** [comment]
> No Cash/license update when you purchase license.

Takes cash and updates in database, but does not give license. 

> Press escape to get rid of the annoying storagebox.

Working as intended. 

> Confirmation before selling vehicles out of the garage.

Working as intended. 

Could the vehicle's display name be used rather than the classname? 

Could the sell vehicle dialog box be centred relative to the main garage dialog box? 
For reference, here it is currently:

**Pull request closed without merging**